expo/gbe/report_views.py

```diff
@@ -165,14 +181,30 @@ def env_stuff(request, conference_choice=None):
 def personal_schedule(request, profile_id='All'):
     viewer_profile = validate_perms(request, 'any', require=True)
 
+    conference_slugs = conf.Conference.all_slugs()
+    if request.GET and request.GET.get('conf_slug'):
+        conference = conf.Conference.by_slug(request.GET['conf_slug'])
+    else:
+        conference = conf.Conference.current_conf()
+
     if profile_id == 'All':
         people = conf.Profile.objects.all().select_related()
     else:
-        people = []
+        people = []  #  Set it to be self, in list format
+
+    tmp_peeps = []
```

Comment:
These variable names are not acceptable. "people", "person" and "event" are the words we're looking for here.